sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -1,16 +1,10 @@
 # Release History
 
-## 1.28.0-beta.1 (Unreleased)
-
-### Features Added
```

Comment:
Should we mention any of the other changes or fixes that went in in this release?

Reply:
I don't see much other than https://github.com/Azure/azure-sdk-for-net/pull/33582 which affect Core itself. Are there any other specific changes you had in mind?

Reply:
@JoshLove-msft fixed a distributed tracing bug in this PR as well: https://github.com/Azure/azure-sdk-for-net/pull/33716

Reply:
That's in shared source.

Reply:
Ah, got it.  Do we not document changes to shared source in CHANGELOGs anywhere?

Reply:
It's a good question - technically, it would need to be documented in any library that is affected by the change. Perhaps in this case it would make sense to mention it in Core, although others are also specifically affected. 

@JoshLove-msft  Would you mind writing up a bullet for this change? I can incorporate it into the PR.

Reply:
I would leave the tracing one out of the Core change logs - it only impacts SB/EH - we can make sure to mention it there.

Reply:
Thanks for checking @annelo-msft!